OrbitQt/OverlayWidget.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "OverlayWidget.h"
+
+namespace OrbitQt {
+
+OverlayWidget::OverlayWidget(QWidget* parent)
+    : QWidget(parent), parent_(parent), ui_(std::make_unique<Ui::OverlayWidget>()) {
+  CHECK(parent != nullptr);
+  parent->installEventFilter(this);
+  ui_->setupUi(this);
+  ui_->cancelButton->setEnabled(true);
+}
+
+void OverlayWidget::paintEvent(QPaintEvent* /*event*/) {
+  QPainter(this).fillRect(rect(), {100, 100, 100, 128});
```

Comment:
Where does this value come from? It's a color isn't it? Could you put that into a const variable?

Reply:
It comes from thin air ;) I added a constant for it